### PR TITLE
fixed: index github link

### PR DESCRIPTION
**Pull request by liuyepiaoxiang**

### CODE REVIEW

File: .replit [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+[gitHubImport]
+requiredFiles = [".replit", "replit.nix", ".config"]
+
+[languages]
+
+[languages.typescript]
+pattern = "**/{*.ts,*.js,*.tsx,*.jsx}"
+
+[languages.typescript.languageServer]
+start = "typescript-language-server --stdio"
```
Comment (CGQAQ) [inline .replit:29]: What's this for

File: replit.nix [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{ pkgs }: {
+    deps = [
+        pkgs.yarn
+        pkgs.esbuild
+        pkgs.nodejs-16_x
+
+        pkgs.nodePackages.typescript
+        pkgs.nodePackages.typescript-language-server
+    ];
+}
```
Comment (CGQAQ) [inline replit.nix:10]: and this?

**Review by CGQAQ — CHANGES_REQUESTED** [review]
What's these for

**Comment by phodal** [comment]
这个优先级比较高，我先修了，close 了这个。

**Pull request closed without merging**